src/context/AuthContext.js

```diff
@@ -0,0 +1,99 @@
+import React, {useState, useEffect, createContext} from "react";
+import {useHistory} from "react-router-dom";
+import jwt_decode from "jwt-decode";
+import axios from "axios";
+
+export const AuthContext = createContext({});
+
+const AuthContextProvider = ({children}) => {
+    const history = useHistory();
+
+    const [Auth, toggleAuth] = useState({
```

Comment:
Een kleinigheidje maar nu Auth een object is waarin meerdere waarden veranderd kunnen worden zou ik de methode setAuth noemen ipv toggle.